deploy/example_catalog/docs/cdh632-readme.md

```diff
@@ -117,4 +117,14 @@ After changing the member count of  the workers and reapplying the cluster,check
 
 #### FUTURE WORK
 
-Adding Kerberos support
\ No newline at end of file
+Adding Kerberos support
+
+
+#### Docker Image location
+
+* docker.io/bluedata/cdh632multi
+
+#### Docker Pull command
+
+* docker pull docker.io/bluedata/cdh632multi:1.4
```

Comment:
did we fix something in 1.4 tag?

Reply:
or is it just documentation

Reply:
@kmathur2 The change is only in the documentation with the latest tag of the cdh image which is 1.4.